Lib/defconQt/featureTextEditor.py

```diff
@@ -17,11 +17,11 @@ def __init__(self, font=None, parent=None):
         self.editor = FeatureTextEditor(self.font.features.text, self)
         self.resize(600, 500)
 
-        fileMenu = QMenu("&File", self)
-        fileMenu.addAction("&Save…", self.save, QKeySequence.Save)
+        fileMenu = QMenu(self.tr("&File"), self)
+        fileMenu.addAction(self.tr("&Save") + "…", self.save, QKeySequence.Save)
```

Comment:
Confused as in, it finds the strings and extracts them correctly, but on a subsequent run, it puts the string "Save?" insead of "Save…" into the translation file, invalidating its' translations.